docs/docs/concepts/memory.md

```diff
@@ -48,6 +56,9 @@ Summarizing messages
 https://github.com/langchain-ai/langchain-academy/blob/main/module-2/chatbot-summarization.ipynb
 https://python.langchain.com/docs/how_to/chatbots_memory/#summary-memory
 
+Storing and retrieving facts or observations from a database during a conversation
+https://github.com/langchain-ai/langgraph/pull/1850
```

Comment:
@rlancemartin I put a link to the PR now, but when this is landed and published we should update it.

Reply:
thanks!